frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -177,7 +179,7 @@ end
 
 -- Returns extended and customized metadata.
 function BookInfo.extendProps(original_props, filepath)
-    local custom_metadata_file = DocSettings:getCustomMetadataFile(filepath)
+    local custom_metadata_file = filepath and DocSettings:getCustomMetadataFile(filepath)
```

Comment:
Is this just for safety ? Or are these cases when it is called without a filepath?

Reply:
If we call without `filepath`, do not customize.
We do it in covermenu, where `bookinfo` is already customized.

Reply:
> Then, why does covermenu need to have it go thru :extendProps() ?

To make a copy of `bookinfo` fileds that are needed in `book_props`.

Reply:
Is it possible that the data in coverbrowser is obsolete and so can't be trusted?
ie. if customizing when in file browser in classic mode - then later switching to detailed list mode ?
(Not saying we should do something about that case - just wondering how trustworthy the book_props coming from SQL is. I guess I will have the same issue for statistics if they are disabled and re-enabled... Also, dunno if people use detailed/mosaic in history, and classic mode in File browser.)

Reply:
When customizing, we refresh (delete) coverbrowser cache entry even in Classic mode.
So `bookinfo` is up to date.
An extreme case if to disable coverbrowser plugin, customize, then enable the plugin back.